wiki_articles/strict_weak.md

```diff
@@ -0,0 +1,29 @@
+# What Is a Strict Weak Ordering in C++?
+
+Algorithms such as
+**[std::sort](https://en.cppreference.com/w/cpp/algorithm/sort)** and
+**[std::set](https://en.cppreference.com/w/cpp/container/map)** require a
+*[Compare](https://en.cppreference.com/w/cpp/named_req/Compare)* function,
+which induces a *[strict weak ordering](https://en.wikipedia.org/wiki/Weak_ordering#Strict_weak_orderings)*.
+This means that the comparison `comp(x, y)` has to behave similar to `x < y`:
+1. `x < x` is false (*Irreflexivity*) (this makes it *strict*)
+2. `x < y && y < z` implies `x < z` (*Transitivity*)
+3. `x < y` implies `!(y < x)` (*Asymmetry*)
+4. (see below)
+
+## 4. Transitivity of Equivalence
+Elements are allowed to be equivalent, i.e. `equiv(x, y)` is similar to `x == y`:
+- let `equiv(x, y)` be `!(x < y) && !(y < x)`
+- then `equiv(x, y) && equiv(y, z)` implies `equiv(x, z)` (*Transitivity*)
+
+For example, when sorting strings by length, all strings of the same length are
+equivalent to only each other.
+This is okay.
+
+It is **not okay** to have elements like `NaN`, which are equivalent to
+`1` and `2`, but `1` and `2` are not equivalent to each other.
```

Comment:
I think it'd be helpful to clarify what it means for a NaN to be equivalent to 1 and 2 here

Reply:
Done.